config.analysis

```diff
@@ -41,6 +48,9 @@ climo_yr1 = 6
 climo_yr2 = 10
 # the offset year to be added to simulation years
 yr_offset = 1849
+# start and end date for the analysis
+analysis_start_date = 0001-01-01_00:00:00
+analysis_end_date = 9999-01-01_00:00:00
```

Comment:
Thanks, @milenaveneziani.  I can definitely make these changes.  Before I do, I just want to make sure you don't foresee the need to specify start and end times that are not at year boundaries.  I guess I see this restriction as appropriate for ACME but rather limiting for the MPAS-Analysis when it's used more generally.

I guess what I'll do is go along with this plan for this PR, but it's something I think we should rethink in the future.